src/sagemaker/lineage/artifact.py

```diff
@@ -211,6 +212,8 @@ def _get_trial_from_trial_component(self, trial_component_arns: list) -> List:
             # no outgoing associations for this artifact
             return []
 
+        # TODO: remove all imports from smexperiment sdk in a separate change.
+        # Instead, import trial_component and search_expression defined in this main sdk
         get_module("smexperiments")
```

Comment:
nit this probably isn't actually used by experiments plus code but still should be removed

Reply:
Which part do you suggest to remove? The TODO notes?  Will do

Reply:
Removed